Assets/Scripts/Loop/LoopController.cs

```diff
@@ -0,0 +1,44 @@
+/******************************************************************
+*    Author: Elijah Vroman
+*    Contributors: Elijah Vroman
+*    Date Created: 5/30/24?
+*    Description: Not even sure what this needs to do, but heres my best guess
+*******************************************************************/
+
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class LoopController : MonoBehaviour
+{
+    private Timer loopTimer;
+    [SerializeField] private int loopTimerTime;
+    public int LoopTimerTimer => loopTimerTime;
```

Comment:
Can we use a more specific name than looptimertimer and looptimertime?

Reply:
Can we restrict this variables scope so it can only be privately set?

Reply:
> Can we restrict this variables scope so it can only be privately set?

I could be wrong since Im new to that way of declaring variables but Im pretty sure that cannot be accessed by anything since its like "pointing" to the private serialized variable. If we want to make loopTimeAmount not editable in inspector we can ofc do that too.